certs/README.md

```diff
@@ -0,0 +1,26 @@
+# Self-signed certificates
+
+_For development environment only_
+
+The VA uses self-signed certificates with a non-globally trusted Root Certificate
+authority. 
+
+
+Used in src/site/stages/build/drupal/api.js line 54 if using the SOCKS proxy:
```

Comment:
What do you think about linking directly to the file GitHub?

```suggestion
Used in [the Drupal client](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/stages/build/drupal/api.js)'s `proxyFetch` function when using the SOCKS proxy.
```

It's kind of weird referencing a specific line, though, since this will be out of date when the code gets moved around as changes are made. I recommend not referencing a specific line.